database/migrations/2017_07_30_205523_create_reciters_table.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class CreateRecitersTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('reciters', function (Blueprint $table) {
+            $table->increments('id');
+            $table->string('name');
+            $table->text('description');
+            $table->integer('hits');
+            $table->string('image_path');
+            $table->smallInteger('status')->default(0);
+            $table->dateTime('moderated_at')->nullable();
+            $table->integer('moderated_by')->nullable()->unsigned();
```

Comment:
Also, I think we should talk some more about what `moderated_at` and `moderated_by` are being used for and whether or not this is a good option.